### PR TITLE
Blocks registering with a kthais mail account

### LINKED ISSUE #254: Prevent registering with @kthais.com email domain
A common misunderstanding among new members signing up is that they should register a new xxx@kthais.com email. This email doesn't exist and so the email is returned to me.

Can we block that in the form? So you get an error or warning when trying to register with a kthais.com email.

Small fix that makes a difference.

### PR DESCRIPTION
**Pull request by danielbenedi6**
When checking if register, it will block if the email ends with "@kthais.com", so it will not allow to register with this email. This closes #254

### CODE REVIEW

File: app/views.py [hunk 1]
```diff
@@ -87,7 +87,7 @@ def files(request, file_):
 @require_POST
 @csrf_exempt
 def deploy(request):
-    forwarded_for = u"{}".format(request.META.get("HTTP_X_FORWARDED_FOR"))
+    forwarded_for = "{}".format(request.META.get("HTTP_X_FORWARDED_FOR"))
```
Comment (oriolclosa) [inline app/views.py:90]: Why's that?

File: user/views.py [hunk 2]
```diff
@@ -186,6 +186,11 @@ def user_register(request):
                 city, country = city_and_country
         if email:
             email = email.lower()
```
Comment (oriolclosa) [inline user/views.py:188]: Can you `.strip()` as well?

**Review by oriolclosa — CHANGES_REQUESTED** [review]

File: app/views.py (hunk 1 above)
Comment (danielbenedi6) [reply app/views.py:90]: I don't know. I haven't changed that. I just used black and I got that. Maybe it was a previous error.

Comment (danielbenedi6) [reply app/views.py:90]: Okay, upon checking. This is legacy. In python 2.x, you have to put u before a string to indicate it is Unicode, but in python 3.x this is no longer necessary

**Review by danielbenedi6 — COMMENTED** [review]
Requested changes done

**Review by oriolclosa — APPROVED** [review]

**Pull request merged**